Microsoft.VisualStudio.CodeTools/CodeTools14.sln

```diff
@@ -0,0 +1,431 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
:exclamation: This support needs to be implemented without adding new solution files. Currently there are two solutions in the directory; that is already one too many.